solutions/bronze/usaco-989.mdx

```diff
@@ -206,7 +207,34 @@ with open("race.out", "w") as fout:
 </LanguageSection>
 </Spoiler>
 
+<Spoiler title="Hint 1">
+
+What's the best strategy that Bessie could use if, hypothetically, there wasn't a speed limit at the finish line?
+
+</Spoiler>
+
+<Spoiler title="Hint 1 (answer)">
+
+We'd want to increase our speed as much as we can!
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+How can we use this strategy when there **is** a speed limit?
+
+</Spoiler>
+
+<Spoiler title="Hint 2 (answer)">
 
+We'll have Bessie continuously speed up until she's forced to slow down because of the speed limit.
+This strategy works because we've observed that when possible, Bessie should always speed up!
```

Comment:
this is way easier said than done...

Reply:
not really sure what else to say...
😭 

maybe write out a little of the code structure? i feel like it gives it away too much, but lmk what u think

Reply:
ok that's fair